<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMenu</class>
 <widget class="QWidget" name="StudentMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentMenu</string>
  </property>
  <widget class="QPushButton" name="EntryButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>1.录入学生成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>2.删除学生成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReviseButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>3.修改学生成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisPlayButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>4.显示全部学生信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculationButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>5.计算平均成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>6.查询学生信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RankButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>7.按单科成绩排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>学生信息管理系统菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>931</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../QtGuiApplication/QtGuiApplication1/normal background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>EntryButton</zorder>
  <zorder>DeleteButton</zorder>
  <zorder>ReviseButton</zorder>
  <zorder>DisPlayButton</zorder>
  <zorder>CalculationButton</zorder>
  <zorder>SearchButton</zorder>
  <zorder>RankButton</zorder>
  <zorder>Exit</zorder>
  <zorder>label_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
